woocommerce/woocommerce-gutenberg-products-block
# Fix Product categories, Product Tags & Keyword filter not working in Products block

This PR fix [this issue](https://github.com/woocommerce/woocommerce-blocks/issues/8249), according to which following filters set from Inspector Controls for Products block aren't working:
- Product Categories
- Product tags
- Keyword




Fixes woocommerce/woocommerce-blocks#8249 




### Testing



1. Create a new post
2. Add Products block inside Editor
3. Add Product categories and/or Product Tags and/or Keyword to Filters(see screenshot)
![image](https://user-images.githubusercontent.com/16707866/216572384-91a9802b-e598-4b18-bdbc-158ff0d465f9.png)
4. Check the front-end. Only products with added categories, tags, or keywords are showing.


### WooCommerce Visibility




### Changelog

> Fix Product categories, Product Tags & Keyword filter not working in Products block
